src/pages/commits/ActiveCommits/buildVariantCard/WaterfallTaskStatusIcon.test.tsx

```diff
@@ -0,0 +1,78 @@
+import { MockedProvider } from "@apollo/client/testing";
+import userEvent from "@testing-library/user-event";
+import { GET_FAILED_TASK_STATUS_ICON_TOOLTIP } from "gql/queries";
+import {
+  renderWithRouterMatch as render,
+  waitFor,
+} from "test_utils/test-utils";
+import { WaterfallTaskStatusIcon } from "./WaterfallTaskStatusIcon";
+
+const props = {
+  displayName: "multiversion",
+  timeTaken: 2754729,
+  taskId: "task",
+};
+
+const Content = (status: string) => () => (
+  <MockedProvider mocks={[getTooltipQueryMock]} addTypename={false}>
+    <WaterfallTaskStatusIcon {...props} status={status} />
+  </MockedProvider>
+);
+
+test("Tooltip should contain task name, duration and list of failing test names", async () => {
+  const { queryByDataCy, queryByText } = render(Content("failed"), {
+    route: "/commits/evergreen",
+  });
+  userEvent.hover(queryByDataCy("waterfall-task-status-icon"));
+  await waitFor(() => {
+    expect(
+      queryByDataCy("waterfall-task-status-icon-tooltip")
+    ).toBeInTheDocument();
+  });
+  await waitFor(() => {
+    expect(queryByDataCy("waterfall-task-status-icon-tooltip")).toBeVisible();
+  });
```

Comment:
nit. These can likely be replaced by the latter check. Since its more important that the tooltip is visible to the user than it being present in the dom.